packages/vuetify/src/components/VTextarea/VTextarea.sass

```diff
@@ -8,18 +8,8 @@
 
 .v-textarea
   .v-field__input
-    align-self: stretch
-    flex: 1 1 auto
-    outline: none
     mask-image: linear-gradient(to bottom, transparent, transparent calc(var(--v-field-padding-top, 0) / 1.25), black var(--v-field-padding-top, 0))
 
-    $root: &
-
-    @at-root
-      @include tools.density('v-input', $input-density) using ($modifier)
-        @at-root #{selector.append(&, $root)}
-          line-height: $textarea-line-height + math.div($modifier * .125, settings.$spacer) * 1.5
```

Comment:
This shouldn't really be removed, without it density doesn't really do much to multi-line inputs.

Reply:
I don't think textarea line-height should change with density.